core/src/avm2/globals/flash/net/URLRequest.as

```diff
@@ -42,5 +45,18 @@ package flash.net {
 		public function get contentType():String {
 			return this._contentType;
 		}
+
+        public function get requestHeaders():Array
```

Comment:
I think the getter and setter should be indented.

Reply:
Reolved in latest commit